source/index.html

```diff
@@ -978,6 +1080,42 @@ <h3 class='title'>Pricing</h3>
 					</p>
 				</div>
 			</div>
+
+                                <div id='addonchat-content' class='closed'>
+				<div class='pricing-specs'>
+					<div class='amount'>$100</div>
+					<div class='per'>Per</div>
+					<div class='unit'>Month</div>
+					<div class='freebie'>(Chat)</div>
+				</div>
+				<div class='copy'>
+					<p>
+						This is a 100 USD monthly add-on to your bill for every additional chat support platform (e.g Telegram, Intercom).
+					</p>
+					<br/>
+					<p>
+						The added monthly cost still includes 24×7 quality support service (minus 1 hour lunch breaks in certain cases and catastrophic events). Excess questions will still cost an additional $3 per question/case.
+                                        </p>
+				</div>
+			        </div>
+
+                               <div id='addonnonchat-content' class='closed'>
+				<div class='pricing-specs'>
+					<div class='amount'>$200</div>
+					<div class='per'>Per</div>
+					<div class='unit'>Month</div>
+					<div class='freebie'>(Non-chat)</div>
+				</div>
+				<div class='copy'>
+					<p>
+						This is a 200 USD monthly add-on to your bill for each additional non-chat support platform (e.g Email).
+					</p>
+					<br/>
+					<p>
+						The added monthly cost still includes 24×7 quality support service (minus 1 hour lunch breaks in certain cases and catastrophic events). Overage per resolved case will be an additional $5.
+                                        </p>
+				</div>
+			        </div>                 
```

Comment:
There shouldn't be extraneous trailing whitespace on any lines either.  Many editors have a feature that automatically fixes this for you.
```suggestion
			</div>
```